internal/controller/progressive/progressive.go

```diff
@@ -486,7 +486,6 @@ func checkForUpgradeReplacement(
 		}
 
 		if differentFromPromoted {
-			numaLogger.WithValues("old child", existingUpgradingChildDef.GetName(), "new child", newUpgradingChildDef.GetName()).Debug("replacing 'upgrading' child")
```

Comment:
The `newUpgradingChildDef` actually doesn't have the right name when logged up here (due to the way it's created)